frontend/public/components/custom-resource-definition.tsx

```diff
@@ -1,13 +1,22 @@
 import * as _ from 'lodash-es';
 import * as React from 'react';
-import { Link } from 'react-router-dom';
 
-import { ColHead, List, ListHeader, ListPage } from './factory';
-import { Kebab, ResourceKebab, ResourceIcon } from './utils';
+import { ColHead, DetailsPage, List, ListHeader, ListPage } from './factory';
+import { Kebab, navFactory, ResourceKebab, ResourceLink, ResourceSummary, SectionHeading } from './utils';
 import { referenceForCRD } from '../module/k8s';
 
 const { common } = Kebab.factory;
-const menuActions = [...common];
+
+const href = obj => _.get(obj, 'spec.scope') === 'Namespaced'
```

Comment:
I’d give this a more specific name like `crdInstancesPath`

```suggestion
const href = crdInstancesPath => _.get(crd, 'spec.scope') === 'Namespaced'
```